projects.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Projects | Autoslug</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Orbitron:wght@500&family=Sometype+Mono:wght@700&display=swap" rel="stylesheet">
+    <link rel="icon" href="_assets/logo.png">
+</head>
+<body style="margin: 0; display: flex; flex-direction: column; height: 100vh;">
+    <!-- Top bar section -->
+    <div class="topTile">
+        <div class="topbar">
+            <div class="topbarLeft">
+                <img src="_assets/logo.png" width="100" height="100" alt="AutoSlug" style="padding-left: 10px">
+            </div>
+            <div class="topbarRight">
+                <a href="index.html" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none">HOME</a>
+                <a href="about.html" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none">ABOUT</a>
+                <a href="projects.html" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none">PROJECTS</a>
+                <a href="team.html" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none">TEAM</a>
+                <a href="https://secure.ucsc.edu/s/1069/bp18/interior.aspx?sid=1069&gid=1001&pgid=780&cid=1749&dids=1207" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none" target="_blank">DONATE</a>
+            </div>
+        </div>
+    </div>
+    <!-- Page Title -->
+    <div class="title" style="background-color:rgb(72, 9, 116);padding-top: 20px;">OUR PROJECTS</div>
+    <div class="centerTile" style="background-color:rgb(72, 9, 116); flex: 1 0 auto; display: flex; flex-direction: column; justify-content: center;">
+        <!-- Project 1 -->
+        <div class="midbox" style="padding: 40px; display: flex;">
+            <img src="_assets/IMG_3494.PNG" width="700px" height="400px" style="padding-right: 30px;">
```

Comment:
can you crop the bottom few pixels off of this image to get rid of the weird black/gray bars?